llvmlite/tests/test_ir.py

```diff
@@ -202,12 +202,18 @@ def test_metadata_2(self):
         mod = self.module()
         mod.add_metadata([ir.Constant(ir.IntType(32), 123)])
         mod.add_metadata([ir.Constant(ir.IntType(32), 321),
-                          ir.MetaDataString(mod, "kernel")])
+                          ir.MetaDataString(mod, b"kernel")])
```

Comment:
If `MetaDataString` starts accepting both bytes and unicode, we should probably test both here.